src/index.html

```diff
@@ -28,7 +28,7 @@
 
     <script>
       var config = {
-        apiEndpoint: 'http://localhost:8000',
+        apiEndpoint: 'https://api.g8s.godsmack.westeurope.azure.gigantic.io',
```

Comment:
@axbarsan That wasn't intended, was it?

Reply:
oh 💩

Reply:
I think I just commented an outdated commit. Looks fine in the final version. Never mind.

Reply:
Just checked as well, now

Scared me to death haha, great way to kickstart the day 👍